pkg/cover/report.go

```diff
@@ -123,6 +129,17 @@ func (rg *ReportGenerator) prepareFileMap(progs []Prog) (map[string]*file, error
 	if !matchedPC {
 		return nil, fmt.Errorf("coverage doesn't match any coverage callbacks")
 	}
+	if len(allProgPCs) != len(matchedProgPCs) {
+		for mismatch := range allProgPCs {
+			if !matchedProgPCs[mismatch] {
+				fmt.Printf("matchedProgPCs: %v\n", matchedProgPCs)
```

Comment:
Remove prints.
They won't print all mismatching PCs anyway.
We could include some unmatched into the error message. But to be useful on its own it would need to include more info about modules, etc. So some actual values probably won't be useful on its own.

Reply:
Fixed.